playbooks/new-tenant-juju/cleanup.yaml

```diff
@@ -0,0 +1,8 @@
+- hosts: all
+  tasks:
+  - name: 'create tenant name'
+    set_fact:
+      tenant_project_name: "zosci-test-{{ ansible_date_time.date }}-{{ zuul.job }}-{{ zuul.project.short_name }}-{{ zuul-build }}"
```

Comment:
will `ansible_date_time.date` remain the same between `pre` and `cleanup`? what if the job starts shortly before midnight?